js/src/evaluation/_runner.ts

```diff
@@ -64,6 +62,18 @@ type DeprecatedAsyncSummaryEvaluator = (
   examples: Array<Example>
 ) => Promise<EvaluationResult | EvaluationResults>;
 
+type SyncSummaryEvaluator = (
+  runs: Array<Run>,
+  examples: Array<Example>,
+  evaluationResults: Array<ExperimentResultRow>
+) => EvaluationResult | EvaluationResults;
+
+type AsyncSummaryEvaluator = (
```

Comment:
Why is this different from `SummaryEvaluatorT`?

Reply:
This is what we wrap the evaluator to internally, I do not believe it is user facing.